reddit/subreddit.py

```diff
@@ -40,8 +41,9 @@ def get_subreddit_threads(POST_ID: str):
             check_for_async=False,
         )
     except ResponseException as e:
-        if e.response.status_code == 401:
-            print("Invalid credentials - please check them in config.toml")
+        match e.response.status_code:
+            case 401:
+                print("Invalid credentials - please check them in config.toml")
```

Comment:
Can you revert this change? We removed the match statement because it's only supported by >Python 3.10